City-of-Helsinki/berth-reservations-admin
# VEN-942 | Fix the bug of refreshing paginated tables & upgrade 'react-table'

## Description :sparkles:
- Upgrade `react-table`
- Prevent side effects when the initial `sortBy` matches the current state

### Closes :no_good_woman:
[**VEN-942:**](https://helsinkisolutionoffice.atlassian.net/browse/VEN-942) Refreshing paginated tables go back to the first page

## Testing :alembic:

### Automated tests :gear:️
- Fixed all relevant snapshots

### Manual testing :construction_worker_man:
- Go to one of the backend paginated tables e.g. Applications List
- Go to the second page
- Refresh the page
- The table should stay and render the second page as expected
